src/command_modules/azure-cli-acs/setup.py

```diff
@@ -31,7 +31,7 @@
 
 DEPENDENCIES = [
     'azure-mgmt-authorization==0.30.0rc6',
-    'azure-mgmt-compute==2.0.0',
```

Comment:
Please cross check Compute SDK usages. There is a chance the ACS commands are using VM specific SDK, say get VM sizes, which means you need to keep it